build/packages-template/bbb-pads/after-install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+case "$1" in
+  configure|upgrade|1|2)
+    TARGET=/usr/local/bigbluebutton/bbb-pads/config/settings.json
+
+    cp /usr/local/bigbluebutton/bbb-pads/config/settings.json.template $TARGET
+
+    if [ -f /usr/share/etherpad-lite/APIKEY.txt ]; then
+      API_KEY=$(cat /usr/share/etherpad-lite/APIKEY.txt)
+      sed -i "s/ETHERPAD_API_KEY/$API_KEY/g" $TARGET
```

Comment:
In the destination file the `ETHERPAD_API_KEY` is not in quoted (and is not valid json), need to double check that it becomes valid json after the substitution